<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsersDialog</class>
 <widget class="QDialog" name="UsersDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Users online</string>
  </property>
  <widget class="QListWidget" name="listWidgetUsers">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>256</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelUsers">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>62</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Charter</family>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Users:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelUsersDetails">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Charter</family>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>User's information:</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>161</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayoutHeaders">
    <item>
     <widget class="QLabel" name="labelManufacterHeader">
      <property name="text">
       <string>Manufacturer:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelTypeHeader">
      <property name="text">
       <string>Type:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelModelHeader">
      <property name="text">
       <string>Model:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>60</y>
     <width>241</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayoutInformation">
    <item>
     <widget class="QLabel" name="labelManufacter">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelType">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelModel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEditDescription">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>220</y>
     <width>331</width>
     <height>151</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDescriptionHeader">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>180</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Description:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfoToUser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>671</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Info label</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
